.github/PULLREQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)
```

Comment:
remove this file and use default templates

Reply:
File deleted